pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -280,43 +290,53 @@ sub _suggest_imunify360 {
 sub _suggest_iav {
     my ($self) = @_;
 
-    if ( !$self->{iav}{installed} && !$self->{iav}{licensed} ) {
+    if ( !$self->{iav}{installed} && !$self->{iavp}{licensed} ) {
```

Comment:
we are inside `_suggest_iav ` should not we only use `$self->{iav}` and not `iavp` here?

Reply:
They are related to each other. We want to suggest IAV install if IAV is not installed and IAVP is not licensed.

If IAVP is licensed we ask them to install IAVP in the _suggest_iavp() function.

Reply:
thanks